pkgs/development/python-modules/highctidh/default.nix

```diff
@@ -0,0 +1,39 @@
+{
+  lib,
+  buildPythonPackage,
+  setuptools,
+  pytestCheckHook,
+  fetchgit,
+}:
+buildPythonPackage rec {
+  pname = "python3Packages.highctidh";
```

Comment:
When I suggested changing the commit and PR names, I didn't mean to change the `pname` as well. Please revert this:

```suggestion
  pname = "highctidh";
```